forge-gui/res/tokenscripts/c_a_powerstone.txt

```diff
@@ -0,0 +1,7 @@
+Name:Powerstone token
+ManaCost:no cost
+Types:Artifact
```

Comment:
Powerstone is new Artifact Subtype

Reply:
Missed that in the release notes. Thanks for the catch 👍